shell.nix

```diff
@@ -38,13 +38,19 @@ let
       ${pnpm}/bin/pnpm install
       ${pnpm}/bin/pnpm run build
     '')
+    (pkgs.writeScriptBin "build-vscode-common" ''
+      #!/usr/bin/env bash
+      set -e
+      cd $(${pkgs.git}/bin/git rev-parse --show-toplevel)
```

Comment:
not sure if installing in the top level is important for the `update-vscode-build-extension` task